pubtools/pulplib/_impl/model/repository/file.py

```diff
@@ -20,3 +23,53 @@ class FileRepository(Repository):
     )
 
     mutable_urls = attr.ib(default=attr.Factory(lambda: ["PULP_MANIFEST"]), type=list)
+
+    def upload_file(self, filename, relative_url=None):
+        """Upload a file to this repository.
+
+        The upload operation includes 4 steps:
+            1. Request an upload id from pulp
+            2. With the requested upload id, upload the file
+            3. After the upload's done, import the uploaded file to repository
+            4. Delete the upload request
+
+        Args:
+            filename (str)
+                Path to a local file to upload.
+
+            relative_url (str)
+                Path that should be used in remote repository
+
+                if omitted, filename will be used.
```

Comment:
Can you please use full sentences with proper punctuation, so it'd be "if omitted" => "If omitted".